test_scripts/Polices/Validation_of_PolicyTables/preloaded_pt_validation_rules_for_required_parameters.lua

```diff
@@ -0,0 +1,113 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- Behavior of SDL during start SDL in case when PreloadedPT has no required parameters
+-- 1. Used preconditions:
+-- Delete files and policy table from previous ignition cycle if any
+-- Do not start default SDL
+-- 2. Performed steps:
+-- Create PreloadedPolicyTable file without one required parameter
+-- Start SDL
+
+-- Requirement summary:
```

Comment:
Please move this section to top. In last approved version of template we start with it https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32